packages/server/graphql/getValidTeamIds.ts

```diff
@@ -0,0 +1,8 @@
+export const getValidTeamIds = (teamIds: null | string[], tms: string[]) => {
+  // the following comments can be removed pending #4070
```

Comment:
+1 what's #4070 referring to? This: https://github.com/ParabolInc/parabol/issues/4070? 

If so, it looks like we can remove the commented lines